### PR TITLE
initial version of CLI readme

### PR DESCRIPTION
**Pull request by rioliu-rh**
/assign @wewang58

### CODE REVIEW

File: oar/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# OAR (Openshift Automatic Release)
+## OAR Commandline Tool (CLI)
+- Install
+```
+clone git@github.com:openshift/release-tests.git
```
Comment (wewang58) [inline oar/README.md:5]: better to git clone xxx, so people can copy it directly

Comment (rioliu-rh) [reply oar/README.md]: done

File: oar/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  create-test-report         Create test report for z-stream release
+  drop-bugs                  Drop bugs from advisories
+  image-consistency-check    Check if images in advisories and payload...
+  image-signed-check         Check payload image is well signed
+  push-to-cdn-staging        Trigger push job for cdn stage targets
+  stage-testing              Trigger stage pipeline test
+  take-ownership             Take ownership for advisory and jira subtasks
+  update-bug-list            Update bug status listed in report, update...
+```
+- Sub command help
```
Comment (wewang58) [inline oar/README.md:46]: s/command/commands

**Comment by wewang58** [comment]
if we add help about how to uninstall oar cli?  so if have new update, users need to install new cli.

**Comment by rioliu-rh** [comment]
> if we add help about how to uninstall oar cli(pip3 uninstall oar)? so if have new update, users need to install new cli.

uninstall is not a problem of oar, currently we don't have version based release. so if we have changes, the user just need to get latest code from the repo.

**Comment by rioliu-rh** [comment]
/retitle OCPQE-17490 initial version of CLI readme

**Pull request merged**